src/handlers/login.ts

```diff
@@ -1,55 +1,59 @@
 // POST
-import * as jwt from 'jsonwebtoken';
+import * as tokenGen from 'jsonwebtoken';
 import { CustomDynamoDB } from "../dynamodb/database";
-import responseHelper from "../helpers/responseHelper"
-import { validatePassword } from "../helpers/validations";
-import { DAY_IN_S } from "../helpers/constants"
+import responseHelper from "../helpers/responseHelper";
+import { validateEmail, validatePassword } from "../helpers/validations";
 
-const usersDB = new CustomDynamoDB('local-bootcamp-bank-api-users', 'email');
+const userDB = new CustomDynamoDB('local-bootcamp-classroom-api-users', 'email')
 
-export const handler = async (event: any) => {
+  
+export const handler = async(event: any) => {
     const body = JSON.parse(event.body);
 
     const {
-        email,
-        password
+        userEmail, 
+        userPassword
     } = body;
-
-    if (!email || email.trim() === '') {
-        return responseHelper(400, 'Email or password not provided');
+    
+    if(!userEmail || userEmail.trim()==='') {
+            return responseHelper(400, 'Email not provided')
     }
 
-    const user = await usersDB.getItem(email);
+    const emailVerified = validateEmail(userEmail)
 
-    if (!user) {
-        return responseHelper(404, 'User Not Found');
+    if(!emailVerified) {
+        return responseHelper(404, 'Email not verified')     
```

Comment:
Podríamos tener un mejor mensaje de error.
Aquí le decimos Email Not verified, pero significa que no viene en la estructura que nosotros lo pedimos. Quizás pudiera ser algo como `Email format not valid`